examples/llama.py

```diff
@@ -190,6 +190,10 @@ def load(fn:str):
   else:
     return torch_load(fn)
 
+def fetch_tiny_llama():
+  fetch("https://huggingface.co/TinyLlama/TinyLlama-1.1B-Chat-v1.0/resolve/main/tokenizer.model", name="tokenizer.model")
```

Comment:
Don't use this name, what if you have other llamas?

Reply:
Oh true. Model and tokenizer need names to be recognized, but `subdir` should be sufficient. It mimics the `llama3.py` fetch as well.